opentracing-cdi-example/src/main/java/io/opentracing/contrib/cdi/example/AccountService.java

```diff
@@ -0,0 +1,20 @@
+package io.opentracing.contrib.cdi.example;
+
+import io.opentracing.contrib.cdi.Traced;
+
+import java.util.logging.Logger;
+
+/**
+ * This is a simple synchronous EJB, without any knowledge about span context or other OpenTracing semantics. All it
+ * does is specify an interceptor and it's shown as the child of a parent span.
+ *
+ * @author Juraci Paixão Kröhling
```

Comment:
Doesn't Java defaults to UTF-8? I personally never had problems with that in Java.

About the `@author` tag: I don't mind dropping it, and I certainly don't do it for "credits", but I did find it useful in the past to contact the original author of a given code, to ask for some clarification.